src/desktop/config.coffee

```diff
@@ -28,6 +28,10 @@ module.exports =
   COOKIE_DOMAIN: null
   CRITEO_ARTWORKS_ACCOUNT_NUMBER: '35250'
   CRITEO_AUCTIONS_ACCOUNT_NUMBER: '28539'
+  DD_API_KEY: null
+  DD_APM_ENABLED: null
+  DD_TRACE_AGENT_HOSTNAME: 'localhost'
+  DD_SERVICE_NAME: 'metaphysics'
```

Comment:
fource

Reply:
Good catch!

Also, how are these used? As defaults? The hostname should probably never be `localhost` since that won't work in a containerized environment like kubernetes and there won't be a trace agent running in other envs like development.

Reply:
I think `localhost` is okay since it's overridden on k8s.

Reply:
Yeah, I think at this point it's just a reference that it exists and is relevant